README.md

```diff
@@ -11,6 +11,13 @@ See the section below on extracting Neo and Gas as the private chain in this Doc
 
 You will also need to install and configure your neo client to use this private network, which involves editing the protocol.json file to point the seeds at your docker IP addresses.
 
+**Note:** If you are running Ubuntu on Windows (WSL), refer to this article to install and operate docker from Ubuntu:
```

Comment:
Please move this note below the next note, which is more important (link to the Docker Hub)

Reply:
@metachris Okay, thank you for the feedback.